### PR TITLE
Add support for the new Select Entity

### PR DESCRIPTION
**Pull request by Laccolith**
## Proposed change

Adds support for the new select entity type which provides a dropdown box in the interface for selecting one of the given options.
Would be handy to provide quick ways of modifying the behaviour of apps from the interface.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionalit)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [ ] Local tests pass.
- [ ] There is no commented out code in this PR.
- [ ] The code has been formatted using Black (`make black`)
- [ ] Tests have been added to verify that the new code works.

### CODE REVIEW

**Review by ludeeus — CHANGES_REQUESTED** [review]
<https://github.com/net-daemon/integration/blob/main/hacs.json#L6> and <https://github.com/net-daemon/integration/blob/main/custom_components/netdaemon/const.py#L9> needs to be adjusted to `2021.7.0`

**Comment by helto4real** [comment]
Thanks alot. Sorry for missing this one. Been away on vacation. @ludeeus is this OK to merge now?

File: custom_components/netdaemon/const.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 INTEGRATION_VERSION = "main"
 DOMAIN = "netdaemon"
 NAME = "NetDaemon"
-MINIMUM_HA_VERSION = "2020.12.0"
+MINIMUM_HA_VERSION = "2021.7.0"
```
Comment (helto4real) [inline custom_components/netdaemon/const.py:9]: Not sure we want to make this dependency that soon?

Comment (ludeeus) [reply custom_components/netdaemon/const.py:9]: The select entity was added in 2021.7, when this is merged, it can only be used on 2021.7+

**Review by ludeeus — APPROVED** [review]
Looks good, but we might want to wait since this has a hard dependency in the version of core

File: custom_components/netdaemon/const.py (hunk 1 above)
Comment (helto4real) [reply custom_components/netdaemon/const.py:9]: @Laccolith thank you for this. I will merge it but wait a few versions until we are ready to force our users to 2021.7.

**Comment by ludeeus** [comment]
@Laccolith Can you look at the merge conflicts?

**Comment by helto4real** [comment]
I am merging this now. Think it went enough time passed to allwo 2021.7.0 be the required version,

**Pull request merged**